client/src/components/WorkspacePanels/SamplesPanel/Samples.js

```diff
@@ -35,107 +35,11 @@ function WithLoadingComponent({ isLoading, samples, type }) {
         });
       }
 
-      return (
-        <Draggable
-          draggableId={samples[samp].id}
-          index={index}
-          key={"sampledragkey-" + samples[samp].id}
-        >
-          {(provided) => (
-            <div
-              ref={provided.innerRef}
-              {...provided.draggableProps}
-              {...provided.dragHandleProps}
-              className="button"
-            >
-              <div className="samples-text-div">{samples[samp].filename}</div>
-
-              <canvas
-                className="button-canvas"
-                id={"canvas-" + samples[samp].id}
-              ></canvas>
-            </div>
-          )}
-        </Draggable>
-      );
+      return <Entry trackData={samples[samp]} index={index}></Entry>;
     });
   }
-  return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      xmlnsXlink="http://www.w3.org/1999/xlink"
-      style={{
-        margin: `auto`,
-        background: `none`,
-        display: `block`,
-        shapeRendering: `auto`,
-      }}
-      width="100px"
-      height="100px"
-      viewBox="0 0 100 100"
-      preserveAspectRatio="xMidYMid"
-    >
-      <circle
-        cx="50"
-        cy="50"
-        r="0"
-        fill="none"
-        stroke="#4297de"
-        strokeWidth="5"
-      >
-        <animate
-          attributeName="r"
-          repeatCount="indefinite"
-          dur="1s"
-          values="0;44"
-          keyTimes="0;1"
-          keySplines="0 0.2 0.8 1"
-          calcMode="spline"
-          begin="0s"
-        ></animate>
-        <animate
-          attributeName="opacity"
-          repeatCount="indefinite"
-          dur="1s"
-          values="1;0"
-          keyTimes="0;1"
-          keySplines="0.2 0 0.8 1"
-          calcMode="spline"
-          begin="0s"
-        ></animate>
-      </circle>
-      <circle
-        cx="50"
-        cy="50"
-        r="0"
-        fill="none"
-        stroke="#86eef9"
-        strokeWidth="5"
-      >
-        <animate
-          attributeName="r"
-          repeatCount="indefinite"
-          dur="1s"
-          values="0;44"
-          keyTimes="0;1"
-          keySplines="0 0.2 0.8 1"
-          calcMode="spline"
-          begin="-0.5s"
-        ></animate>
-        <animate
-          attributeName="opacity"
-          repeatCount="indefinite"
-          dur="1s"
-          values="1;0"
-          keyTimes="0;1"
-          keySplines="0.2 0 0.8 1"
-          calcMode="spline"
-          begin="-0.5s"
-        ></animate>
-      </circle>
-    </svg>
-  );
-}
+  return <Loading></Loading>;

```

Comment:
I believe empty components don't need a closing tag! So we should be able to write `<Loading />` only.